### ISSUE TITLE
A small suggestion

### ISSUE THREAD

**Issue by guahunyo** [frankchieng/ComfyUI_MagicClothing#5]
A small suggestion, instead of putting all models in the model folder of comfyui, don’t take your ipa model from your own folder. Take it from the public model, otherwise it will be saved twice, which wastes a lot of space, or even It would be better if you let people fill in the model paths themselves.

**Comment by theguaz** [comment]
totally agree, I have all my  IPAdapter FaceID models in a different folder

**Comment by frankchieng** [comment]
good point,put all of IPAdapterFaceid models under the ComfyUI main path of models/IPAdapter and models/lora accordingly should be fine

**Comment by ManuelMultiverse** [comment]
After installing node , my comfyUI cannot read the main folder checkpoints.

**Comment by frankchieng** [comment]
> After installing node , my comfyUI cannot read the main folder checkpoints.

issue has been solved,just git pull and update the project as well as pay attention to the new model location in README file

**Comment by dareima** [comment]
As far as I understand it, you're not supporting different model paths configured in 'extra_model_paths.yaml', is that correct?

I am using the model folders of my A1111 install in ComfyUI to not have to keep redundant models on my disk. So far, I can't get any of the workflows to work due to:

```
Prompt outputs failed validation: Value not in list: model_path: 'magic_clothing_768_vitonhd_joint.safetensors' not in []
MagicClothing_Generate:
- Value not in list: model_path: 'magic_clothing_768_vitonhd_joint.safetensors' not in []
```

**Comment by frankchieng** [comment]
just download magic_clothing_768_vitonhd_joint.safetensors and place it according to the README,i haven't used extra_model_paths.yaml yet,just assign the folder_paths in global namespace

**Comment by dareima** [comment]
> just download magic_clothing_768_vitonhd_joint.safetensors and place it according to the README,i haven't used extra_model_paths.yaml yet,just assign the folder_paths in global namespace

Appreciate your work and feedback, thank you! I do have the required models in the models/checkpoints folder of ComfyUI but get the above error. I'll need to figure out what the global namespace is.
